CONTRIBUTION.md

```diff
@@ -53,42 +41,102 @@ The format can be described more formally as follows:
-  usable. (see below)
-  - Once the PR is approved, remove the hold by commenting `/hold cancel` to merge the PR
+The subject should be no longer than 50 characters, and the body should be no longer than 80 characters. There are some
+[githooks](https://github.com/jorisroovers/gitlint) you can install that can help enforce these style requirements.
+
+## Opening a pull request
+
+Once you are done with the pre-PR checklist, push your changes and create a pull request!
+
+### Format of the pull request
+
+#### Title
+
+When creating your pull request, you should prefix it with the Jira issue, followed by the subsystem name in brackets. It
+should follow the format
+
+```WINC-<number>: [<subsystem>] <title>```
+
+For example, a docs PR would look like
+
+```WINC-959: [docs] reorganizes readme```
+
+In doing this, you’ll be linking your PR to the Jira ticket for tracking.
+
+In the case of bug fix PRs, this will also automatically transition the associated Jira bug.
+
+- Opening the PR: `ASSIGNED` --> `POST`
+- PR merges: `POST` --> `MODIFIED`
+- `MODIFIED` to the `ON_QA` transition will have to be done manually by the PR author.
+- PR closes without merging, the Jira bug transitions back to NEW status.
+
+If you have no Jira issue or are not a Red Hat employee, your title should simply follow the format ```[<subsystem>] <title>```, with ```Fixes #<issue>``` in the comment. 
+
+### Open as draft
+
+The WMCO team opens pull requests as drafts before they are reviewed in order to prevent the tests from running right away
+[Open your PR as a draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) before requesting reviews.
+
+
+## Merging a pull request
+
+Once your PR is open, you'll need at least two reviewers te get it merged.
+
+### Moving your PR out of draft
+
+Before moving your PR out of draft, you must have at least one /lgtm and one /approve on the PR.
+Once you have both, you are free to move it out of draft status by clicking the "ready for review" button
+in your PR interface. This will let your tests start running, and if they pass, your PR will merge automatically.
+
+### Respond to failing tests
+
+If your PR is experiencing a test failure that you believe to be a flake, feel free to run a retest.
+When retesting, be sure to add a link to the failing test, and a snippet of what caused the failure.
+The format should be
+
+```
+/retest-required
+
+<explanation of the error>
+<reason for retest>
+<prow.ci.openshift.org link> 
+
+```<log snippet of the failure> ```
+```
+
+Once your tests are passing, and your PR has an approval and a LGTM label, the CI should merge it automatically.
 
 ![Sample PR life-cycle](/images/PR-workflow.png)
+
+## Backporting changes
+Things like bugfixes should be backported to all [supported versions](https://access.redhat.com/support/policy/updates/openshift#windows).
```

Comment:
Please remove `Things like`. And what about the non-bugfix case? That needs to be mentioned here also. With non-bugfix PRs, backports can be done from the main PR to the appropriate branches. There is not rule as such regarding which branch to pick and will depend on the PR.